src/tests/ffi.cpp

```diff
@@ -4500,6 +4500,114 @@ TEST_F(rnp_tests, test_ffi_enarmor_dearmor)
     }
 }
 
+TEST_F(rnp_tests, test_ffi_customized_enarmor)
```

Comment:
I would suggest extending functionality of tests with following cases:
- make sure it works on some real data, like exporting large key
- check that export with every available line length in cycle (i.e. from 16 till 76), dearmoring it back.

Reply:
@ni4 key dumps are not covered by `rnp_output_to_armor`. It's rather tricky to propagate this setting (and it wasn't required by the ticket)

Reply:
@rrrooommmaaa No need to propagate this to key export armoring - you can just export raw key to  `rnp_output_to_armor()` with modified line length. That would allow to test real usecase, as it was asked by TB team. It is hard to describe all the details in the issue description, but it is quite logical to add more thorough tests when we adding such new functionality. Say, who knows, whether code works well on 2kb armored message with line length of 50.

Reply:
I have done this and other fixes too.